lib/plugin-manager.js

```diff
@@ -251,55 +265,77 @@ class PluginManager extends EventEmitter {
    * @private
    */
   _registerSourceEvents(source) {
-    source.on('init', () => {
-      Log.info(`Initializing ${source.type} source ${source.name}`, {
-        source: source.name,
-        type: source.type
-      });
+    source.on('init', (logMetadata) => {
+      Log.log('INFO', `Initializing ${source.type} source ${source.name}`,
+        Object.assign({
+          sourceName: source.name,
+          sourceType: source.type
+        },
+        logMetadata));
     });
 
-    source.on('startup', () => {
-      Log.info(`${source.name} started up successfully.`);
+    source.on('startup', (logMetadata) => {
```

Comment:
The `init` and `startup` events are fired from the same closures in all source modules. We should pick one and remove the other.